<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InfoEventStackWidget</class>
 <widget class="QWidget" name="InfoEventStackWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>235</width>
    <height>128</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>4</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>4</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QToolButton" name="previousButton">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="application.qrc">
         <normaloff>:/images/go-previous.png</normaloff>:/images/go-previous.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="pageLabel">
       <property name="text">
        <string>0 / 0</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="nextButton">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="application.qrc">
         <normaloff>:/images/go-next.png</normaloff>:/images/go-next.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QToolButton" name="closeButton">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="application.qrc">
         <normaloff>:/images/application-exit.png</normaloff>:/images/application-exit.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QStackedWidget" name="stackedWidget"/>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="application.qrc"/>
 </resources>
 <connections/>
</ui>
